tests/abstract_model_inheritance/tests.py

```diff
@@ -0,0 +1,320 @@
+from __future__ import unicode_literals
```

Comment:
Instead of a new test app, I'd favor `tests/model_inheritance/test_abstract_inheritance.py`